src/devices/Tcs3472x/Registers.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Tcs3472x
+{
+    [Flags]
```

Comment:
do you combine those that `[Flags]` is useful? feels kinda unusual for registers :smile:

Reply:
Correct, I used to but changed the architecture. Forgot to remove the [Flags] :-)